### ISSUE TITLE
[Feature request] send invite link

### ISSUE THREAD

**Issue by wuast94** [mailcow/mailcow-dockerized#3173]
Would be nice to have a function to add a person to the mail server

For example: I have a friend who wants an email Adress on my server. As far as I know I must create the mailbox and send him his mail Adress and password. 

Would be nicer if I can send an invite link to his email Adress where I can assign an email Adress and he can create his login for that.

**Comment by andryyy** [comment]
Indeed a good idea. 

We shouldn't send passwords though. Or a one time password. :)

**Comment by JustAB0x** [comment]
@andryyy if i get his idea right:
1. Admin creates invite with fixed email address
2. Invite is send to User (Copy URL and send it manually or send to email)
3. User sets display Name & Password
4. Login is active and email is usable

did i get that correct @wuast94 ?

**Comment by wuast94** [comment]
> @andryyy if i get his idea right:
> 
> 1. Admin creates invite with fixed email address
> 2. Invite is send to User (Copy URL and send it manually or send to email)
> 3. User sets display Name & Password
> 4. Login is active and email is usable
> 
> did i get that correct @wuast94 ?

yeah that's right :) for better control we can add settings for this invite link too. how long is it valid. max space for that account etc.

**Comment by dragoangel** [comment]
As additional possible enhancement it would be cool if Mailcow will support external recovery email in addition to invite link & allow send invite link directly to recovery email, what you think @andryyy ? This will allow users to reset their password from their own emails via temporary links. Of course this optional & can be restricted via existings ACL per user

**Comment by andryyy** [comment]
I will try to work on it soon. Very, very busy at the moment. :( I like your ideas, though. :) Should be implemented next.

**Comment by vermium** [comment]
I like this idea too

**Comment by youth-foundation-bw** [comment]
Hi @andryyy,

Has anything come out of this idea? We also need a way for our project to invite people using an alternative email address. It should be as simple as possible to invite a large number of users without much effort for the admin. And of course, it should be a secure process for transmitting the credentials. We are also willing to pay for this development work.

Best regards,
Paul

**Comment by xeruf** [comment]
would love this as well!
coming from HestiaCP where you can, at mailbox creation time, setup aliases, a forwarding address (so the person does not have to bother with any setup if it is just an alias for them) and an invitational email with setup instructions including IMAP configuration etc

**Comment by xeruf** [comment]
andryyy has departed and development has been slow recently, want to look into whether I can contribute a bit

any funding opportunities still open or did you find other solutions (especially @youth-foundation-bw ?)
